consoleApp/src/main/java/com/denis/consoleapp/service/Switch.java

```diff
@@ -0,0 +1,38 @@
+package com.denis.consoleapp.service;
+
+public class Switch {
```

Comment:
i did not get the idea of this class, if you want to organize commands - it is not the best solution. please make a deeper look on command pattern or chain of responsibility pattern